### PR TITLE
retry option in commands

### PR DESCRIPTION
**Pull request by tanishgupta1**

### CODE REVIEW

**Comment by sourabh912** [comment]
@tanishgupta1 : Are your changes merged into master ?

**Comment by tanishgupta1** [comment]
@sourabh912 My changes are not merge to master. Please merge it to master.

File: qds_sdk/commands.py [hunk 1]
```diff
@@ -414,6 +415,7 @@ class SparkCommand(Command):
 
     optparser.add_option("--print-logs", action="store_true", dest="print_logs",
                          default=False, help="Fetch logs and print them to stderr.")
+
```
Comment (harshshah87) [inline qds_sdk/commands.py]: Spark also has a retry option.

**Comment by harshshah87** [comment]
The changes looks good except one comment. Please address the test case for spark as well. Will merge it afterwards.

**Comment by harshshah87** [comment]
All I had to do was merge from the master. This would enable me to resolve the conflict early instead of depending on the github's merge (Happens during the PR merge when travis does the testing).  I will now move it to qubole:unreleased branch.

**Comment by harshshah87** [comment]
Closing the PR.

**Pull request closed without merging**